k8s-e2e-external-storage.groovy

```diff
@@ -152,6 +152,11 @@ node('cico-workspace') {
 				// single-node-k8s.sh pushes the image into minikube
 				podman_pull(ci_registry, "docker.io", rook_ceph_cluster_image - d_io_regex)
 			}
+
+			// busy box is used in external storage testing
+			podman_pull(ci_registry, "docker.io", "library/busybox:1.29")
```

Comment:
should we use tag, `latest`, or `stable` instead of `1.29` specific version?

Reply:
This is the image that gets pulled by kubernetes E2E. AFAIK we don't have the option to specify the image

Reply:
I have pushed this to the CI local registry:
```
$ podman push docker.io/library/busybox:1.29 registry-ceph-csi.apps.ocp.ci.centos.org/library/busybox:1.29
```

Reply:
Thanks @nixpanic